tests/fixtures/analysis_form/analysis_form.json

```diff
@@ -66,39 +62,23 @@
       }
     },
     {
-      "type": "datetime",
-      "name": "record_a_note/acme_timestamp",
```

Comment:
It probably made sense as a forward-looking example, but now that we are implementing qualitative analysis questions, `datetime` is not a supported question type